Src/MoneyFox.Business/MoneyFox.Business.csproj

```diff
@@ -111,6 +111,11 @@
       <Name>MoneyFox.Foundation</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="DefaultLanguageSetter">
+      <HintPath>..\..\DefaultLanguageSetter.dll</HintPath>
```

Comment:
Is the DefaultLanguageSetter a library you used? Isn't there a nuget for it or could we just make one instead of put it to the project folder directly?